assets/js/components/Dashboard/AlertBanner.tsx

```diff
@@ -2,15 +2,19 @@ import React, { ComponentType } from "react";
 import { ArrowRepeat } from "react-bootstrap-icons";
 import { formatEffect, translateRouteID } from "../../util";
 import { useParams } from "react-router-dom";
-import { BannerAlert } from "./Dashboard";
+import { useScreenplayContext } from "../../hooks/useScreenplayContext";
+import { Alert } from "../../models/alert";
 
-interface Props {
-  bannerAlert: BannerAlert;
+interface BannerAlert {
+  alert: Alert;
+  closedAt?: string;
```

Comment:
I _think_ the `closedAt` prop was removed in the banner alert PR. Was it added back intentionally or does the branch just need an update from master?

Reply:
That interface is now used to represent a `BannerAlert` object and not the props for `AlertBanner`. So the props are still gone, but it made more sense to have the `BannerAlert` interface exported from this file instead of `Dashboard`.

Reply:
Ah I see that makes sense